doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1194,6 +1206,11 @@ Response:
                 ]
             },
             "callback":  "http://localhost:1234",
+            "metadata" : {
+                "provider": {
+                   "value": "example.com"
```

Comment:
`type: null` should be included.

(Sorry, this is a post-LGTM comment... I should have looked closer before casting LGTM... :( )